packages/toolpad-app/cli/index.ts

```diff
@@ -26,22 +44,19 @@ interface RunCommandArgs {
 }
 
 const waitForHealthCheck = async (port: number): Promise<void> => {
-  const { default: chalk } = await import('chalk');
   const TOOLPAD_BASE_URL = `http://localhost:${port}/`;
   const checkedUrl = new URL(HEALTH_CHECK_URL, TOOLPAD_BASE_URL);
   for (let i = 1; i <= MAX_RETRIES; i += 1) {
     try {
       // eslint-disable-next-line no-console
-      console.log(`(${i}/${MAX_RETRIES}) trying to reach "${checkedUrl.href}"...`);
+      console.log(`(${i}/${MAX_RETRIES}) trying to reach "${HEALTH_CHECK_URL}"...`);
```

Comment:
```suggestion
      console.log(`(${i}/${MAX_RETRIES}) trying to reach "${HEALTH_CHECK_URL}"...`);
```

I don't think we should log here. The healthcheck is an implementation detail, and shouldn't even kick in.